mopidy/http/handlers.py

```diff
@@ -98,9 +98,7 @@ class WebSocketHandler(tornado.websocket.WebSocketHandler):
     clients = set()
 
     @classmethod
-    def broadcast(cls, msg):
-        loop = tornado.ioloop.IOLoop.current()
-
+    def broadcast(cls, loop, msg):
```

Comment:
On Python 3, I'd consider making `io_loop` a kwarg-only argument, though that is an exercise left for the (near) future. However, we can order the arguments now so that we easily can change this to `def broadcast(self, msg, *, io_loop)` later.

```suggestion
    def broadcast(cls, msg, io_loop):
```